src/audio-worklet/free-queue/src/interface/README.md

```diff
@@ -47,13 +53,11 @@ emcc \
 	-o {output_file_name.js}
 ```
 
-### Example Usage
-
 An example workflow of using interface.
 
-1. Create FreeQueue in C.
-2. Query for Pointers through JS.
-3. Create JS instance of FreeQueue through Pointers.
-4. Now you can push and pull from either side, while one side being producer and other being consumer.
+1. Create a FreeQueue in C.
+2. Query for Pointers of FreeQueue data members through JS.
+3. Create JS instance of FreeQueue using queried Pointers.
+4. Now we can push and pull from either side, while one side being producer and other being consumer.
 
-[FreeQueue WebAssembly Example: DivyamAhuja/free_queue_c](https://github.com/DivyamAhuja/free_queue_c)
\ No newline at end of file
+[FreeQueue WebAssembly Example: DivyamAhuja/free_queue_c](https://github.com/DivyamAhuja/free_queue_c)
```

Comment:
This seems redundant. We'll add examples to this repository in the near future, and no external links are necessary.

Reply:
Okay!